Import/SampleForLabelsInReports/Import/Report/Part Report With Label.xml

```diff
@@ -9,100 +9,101 @@
+				.cellDashedBottom {border-bottom:1px #666666 dashed;}
+				.cellDashedLeft {border-left:1px #666666 dashed;}
+				.cellHeader {background-color:#CCCCCC;border-top:1px #000000 solid;border-right:1px #000000 solid;border-bottom:1px #000000 solid;padding:2px;text-align:center;text-transform:capitalize;text-align:center;font-family:helvetica;font-weight:bold;font-size:8pt;}
+			</style>
+			      <body topmargin="50" leftmargin="50">
+				        <table border="0" cellspacing="0" cellpadding="0" width="950">
+					          <tr>
+						            <td width="150" uniqueID="ms__id2"/>
+						            <td width="200" uniqueID="ms__id3"/>
+						            <td width="150" uniqueID="ms__id4"/>
+						            <td width="150" uniqueID="ms__id5"/>
+						            <td width="150" uniqueID="ms__id6"/>
+					  </tr>
+					          <tr valign="top">
+						            <td align="left" uniqueID="ms__id77" colspan="9">
+							              <img src="../images/aras-innovator.svg" height="40"/>
+						      </td>
+					  </tr>
+					          <tr valign="bottom">
+						            <td colspan="2" style="font-family:helvetica;font-size:15pt;color:#DA1943;padding:2px;" align="left" uniqueID="ms__id79">Part Report (with Make/Buy Labels)</td>
+						            <td colspan="3" style="font-family:helvetica;font-size:10pt;padding:2px;" align="right" uniqueID="ms__id80">
+							            <xsl:value-of select="//Item/timestamp"/>
+							                <xsl:if test="timestamp='' or not(timestamp)">
+								                  <xsl:text/>
+							                  </xsl:if>
+						            </td>
+					  </tr>
+					          <tr valign="top">
+						            <td class="cellHeader" uniqueID="ms__id81" align="left" colspan="" rowspan="" height="" width="" style="border-left:1px #666666 solid;">Part Number</td>
+						            <td class="cellHeader" uniqueID="ms__id82">Name</td>
+						            <td class="cellHeader" uniqueID="ms__id83">Cost</td>
+						            <td class="cellHeader" uniqueID="ms__id84">Make/Buy Value</td>
+						            <td class="cellHeader" uniqueID="ms__id84">Make/Buy Label</td>
+					  </tr>
+					          <xsl:for-each select="//Item">
+						            <xsl:sort select="name" order="ascending" case-order="upper-first" data-type="text"/>
+						            <tr valign="center">
+							              <td style="font-family:helvetica;font-size:8pt;font-weight:bold;border-left:1px #666666 solid;padding:2px;" class="cellSolidRightBottom" align="left" uniqueID="ms__id90" colspan="" rowspan="" height="" width="">
+								                <xsl:value-of select="item_number"/>
+								                <xsl:if test="item_number='' or not(item_number)">
+									                  <xsl:text> </xsl:text>
+								        </xsl:if>
+							      </td>
+							              <td style="font-family:helvetica;font-size:8pt;padding:2px;" class="cellSolidRightBottom" align="left" uniqueID="ms__id91">
```

Comment:
It would be good to create CSS classes for some of these more commonly used styles. As a general rule of thumb, the more separation between style and content, the better.